### DISCUSSION TITLE
sdbus-cpp version problem

### DISCUSSION THREAD

**Discussion by dengxk1234** [end-4/dots-hyprland#891] [Q&A]
Today, the hyprlock project [updated](https://github.com/hyprwm/hyprlock/commit/f48540fcd4421f1227e6e793e54f3f04b9df61d1) its requirements, which means the sdbus-cpp lib in the offical repo(1.5.0) is [oudated](https://archlinux.org/packages/extra/x86_64/sdbus-cpp/)(hyprlock needs 2.0.0). I've tried to build the sdbus-cpp lib from source, but it didn't work(not sure if it's my problem).
Any fixes for this?

**Comment by mambucodev** [comment]
I downloaded the hyprlock from the repos instead of -git for now.

**Comment by dengxk1234** [comment]
It seems that the lib in the offical repo has been updated to 2.0.0 today, so the problem shouldn't appear again.
